task/github-create-deployment/0.2/README.md

```diff
@@ -0,0 +1,67 @@
+## GitHub Create Deployment
+
+The `github-create-deployment` Task lets you create a GitHub deployment.
+
+See GitHub's deployment API on [Create a deployment](https://docs.github.com/rest/reference/repos#create-a-deployment) for more information.
+
+### Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/github-create-deployment/0.2/github-create-deployment.yaml
+```
+
+### Secrets
+
+This Task requires access to a GitHub token set via a Kubernetes Secret. By default, the name of this Secret should be `github` and the secret key should be `token`, but you can configure this via the `GITHUB_TOKEN_SECRET_NAME` and `GITHUB_TOKEN_SECRET_KEY` [parameters](#parameters) described below.
+
+To create such a Secret via `kubectl`:
+
+```
+kubectl create secret generic github --from-literal token="MY_TOKEN"
+```
+
+See GitHub's documentation on [Scopes for OAuth Apps](https://docs.github.com/developers/apps/scopes-for-oauth-apps) to figure out what scopes you need to give to this token to create deployments.
```

Comment:
Instead of letting users figure out themselves what scopes they need probably we can document this here. WDYT?
Like for this task requirements would be
```
Users with repo or repo_deployment scopes can create a deployment for a given ref
```

Reply:
I totally agree with you. Thanks for pointing this out! 👍 🙂

Reply:
I think `repo_deployment` is enough since `repo` contains `repo_deployment` as below, WDYT?

![image](https://user-images.githubusercontent.com/6194958/112434211-a6e39980-8d86-11eb-97e2-e7eb179561dd.png)